SteamcLog/Classes/Helper/LogLevelPreset.swift

```diff
@@ -9,57 +9,55 @@
 import Foundation
 
 public enum LogLevelPreset: String, Codable {
-    /// Disk: verbose, system: verbose, remote: none, analytics: disabled
-    case firehose
-    /// Disk: none, system: debug, remote: none, analytics: disabled
-    case develop
-    /// Disk: verbose, system: none, remote: warn, analytics: enabled
-    case releaseAdvanced
-    /// Disk: none, system: none, remote: warn, analytics: enabled
+
+    case debugVerbose
+    case debug
     case release
+    case releaseAdvanced
 
     var global: LogLevel {
         switch self {
-        case .firehose: return .info
-        case .develop: return .info
-        case .releaseAdvanced: return .info
+        case .debugVerbose: return .info
+        case .debug: return .info
         case .release: return .warn
+        case .releaseAdvanced: return .info
         }
     }
 
-    var sentry: LogLevel {
+    var remote: LogLevel {
         switch self {
-        case .firehose: return .none
-        case .develop: return .none
-        case .releaseAdvanced: return .info
-        case .release: return .warn
+        case .debugVerbose: return .none
+        case .debug: return .none
+        case .release: return .info
+        case .releaseAdvanced: return .warn
```

Comment:
I think this should be info - isn't the intent that `releaseAdvanced` returns more info than `release?`

Reply:
Good catch. Looks like based on the table I pulled from our meeting this should be debug. I'll go through again with a fresh set of eyes and make sure that all lines up properly.

Reply:
Fixed